var/spack/repos/builtin/packages/libzip/package.py

```diff
@@ -14,3 +14,5 @@ class Libzip(AutotoolsPackage):
     url      = "https://nih.at/libzip/libzip-1.2.0.tar.gz"
 
     version('1.2.0', '5c3372ab3a7897295bfefb27f745cf69')
+
+    depends_on('zlib')
```

Comment:
> You'll need zlib (at least version 1.1.2). It comes with most operating systems.

Please add the version limit and if you feel like it you could also some of the optional stuff 

> For supporting bzip2-compressed zip archives, you need bzip2.
 For AES (encryption) support, you need one of these cryptographic libraries, listed in order of preference:
Apple's CommonCrypto (available on macOS and iOS)
GnuTLS
mbed TLS
OpenSSL >= 1.0.
Microsoft Windows Cryptography Framework

https://github.com/nih-at/libzip/blob/master/INSTALL.md